frontend-react/src/components/Admin/EditReceiverSettings.tsx

```diff
@@ -12,6 +12,7 @@ import {
 } from "../../contexts/SessionStorageTools";
 import { jsonSortReplacer } from "../../utils/JsonSortReplacer";
 import Spinner from "../Spinner";
+import { getErrorDetail } from "../../utils/misc";
```

Comment:
Naming: Maybe something more expliciet like `getErrDetailsFromReponse`?

Reply:
@TomNUSDS agreed, it's now getErrorDetailFromResponse